packages/bsync/src/config.ts

```diff
@@ -92,12 +117,16 @@ export const readEnv = (): ServerEnvironment => {
     dbMigrate: envBool('BSYNC_DB_MIGRATE'),
     // secrets
     apiKeys: envList('BSYNC_API_KEYS'),
-    // revenue cat
+    // purchases
     revenueCatV1ApiKey: envStr('BSYNC_REVENUE_CAT_V1_API_KEY'),
     revenueCatV1ApiUrl: envStr('BSYNC_REVENUE_CAT_V1_API_URL'),
     revenueCatWebhookAuthorization: envStr(
       'BSYNC_REVENUE_CAT_WEBHOOK_AUTHORIZATION',
     ),
```

Comment:
This is the full HTTP header—how would you feel about making the configuration just receive the credential itself, the bearer token.

Reply:
Can look to `BSYNC_API_KEYS` for a similar situation.

Reply:
Yeah, I do still lean that way.  (And this does seem like a bearer token to me 👍)